From 1b4e3ab010b04dfe2142e03854b92dc55a2b5a09 Mon Sep 17 00:00:00 2001
From: David Abdurachmanov <david.abdurachmanov@sifive.com>
Date: Tue, 12 Jan 2021 20:36:24 +0200
Subject: [PATCH 26/34] sifive: fu740: fix cache controller signals order

Cache controller in FU740 compared to FU540 has 4 signals instead of 3.
DirFail is the last one. Signals are wired differently in FU740 thus
the change in the order is required.

Signed-off-by: David Abdurachmanov <david.abdurachmanov@sifive.com>
---
 arch/riscv/dts/fu740-c000.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/dts/fu740-c000.dtsi b/arch/riscv/dts/fu740-c000.dtsi
index ee0ab3f..72564c2 100644
--- a/arch/riscv/dts/fu740-c000.dtsi
+++ b/arch/riscv/dts/fu740-c000.dtsi
@@ -271,7 +271,7 @@
 			cache-size = <2097152>;
 			cache-unified;
 			interrupt-parent = <&plic0>;
-			interrupts = <19 20 21 22>;
+			interrupts = <19 21 22 20>;
 			reg = <0x0 0x2010000 0x0 0x1000>;
 		};
 		gpio: gpio@10060000 {
-- 
2.7.4

